build/gen_stub.php

```diff
@@ -83,9 +83,15 @@ function processStubFile(string $stubFile, Context $context, bool $includeOnly =
             }
         }
 
+        /* As exit() and die() used to be proper token/keywords we need to hack-around for versions prior to PHP 8.4 */
```

Comment:
Is this comment still correct?

Reply:
Right, those are still proper tokens actually